### ISSUE TITLE
Including the word "reissue" in a tag name breaks tests

### ISSUE THREAD

**Issue by sarahgoldman** [serenity-js/serenity-js#894]
**Describe the bug**

If I tag a scenario with "@reissue" or even a tag name containing that word, e.g. "@reissueassignment", the tests fail with error:

```
[12:25:46] E/launcher - Error: Error: Tag name should have a property "length" that is greater than 0
    at Object.ensure (C:\jdev2020\git\via-automated-testing\node_modules\tiny-types\src\ensure.ts:31:15)
    at new Tag (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\core\src\model\tags\Tag.ts:23:9)
    at new IssueTag (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\core\src\model\tags\IssueTag.ts:10:9)
    at C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\core\src\model\tags\Tags.ts:17:66
    at Array.map (<anonymous>)
    at MatchesRegExp.transformation (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\core\src\model\tags\Tags.ts:17:53)
    at MatchesRegExp.MatcherRule.execute (C:\jdev2020\git\via-automated-testing\node_modules\tiny-types\src\pattern-matching\rules\MatcherRule.ts:13:21)
    at StringMatcher.PatternMatcher.else (C:\jdev2020\git\via-automated-testing\node_modules\tiny-types\src\pattern-matching\PatternMatcher.ts:17:71)
    at Function.from (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\core\src\model\tags\Tags.ts:18:18)
    at C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\cucumber\src\listeners\messages\parser\CucumberMessagesParser.ts:233:80
    at Array.map (<anonymous>)
    at CucumberMessagesParser.scenarioFrom (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\cucumber\src\listeners\messages\parser\CucumberMessagesParser.ts:233:64)
    at CucumberMessagesParser.parseTestCaseStarted (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\cucumber\src\listeners\messages\parser\CucumberMessagesParser.ts:102:34)
    at EventEmitter.<anonymous> (C:\jdev2020\git\via-automated-testing\node_modules\@serenity-js\cucumber\src\listeners\messages\CucumberMessagesListener.ts:49:41)
    at EventEmitter.emit (events.js:323:22)
    at EventEmitter.emit (domain.js:482:12)
[12:25:46] E/launcher - Process exited with error code 100
```

**To Reproduce**
Tag a scenario with "@reissue" or "@reissuesomething". 

**Note**: I am using with protractor-cucumber-framework@~8.0.0

Here is my feature file:
```
@VIA-150 @establishment @establishmentdetails
Feature: VIA-150 CREATE ASSIGNMENTS
    All functionality related to creating an assignments
    
    @reissue
    Scenario: 150-16449-AC1-HP2
        Given "Coordinator" user is logged in
        And the user is viewing an establishment that is marked to be reissued an assignment
        When the user creates an assignment for the establishment
        Then the system shall display the "Establishment Detail" screen
        And the establishment shall not display the Reissue badge
```

When I change it to "reissu" without the "e", it works as expected:

```
@VIA-150 @establishment @establishmentdetails
Feature: VIA-150 CREATE ASSIGNMENTS
    All functionality related to creating an assignments
    
    @reissu
    Scenario: 150-16449-AC1-HP2
        Given "Coordinator" user is logged in
        And the user is viewing an establishment that is marked to be reissued an assignment
        When the user creates an assignment for the establishment
        Then the system shall display the "Establishment Detail" screen
        And the establishment shall not display the Reissue badge
```

I believe this has to do with [Jira integration](https://serenity-js.org/handbook/integration/jira-and-other-issue-trackers.html) but if I am not putting a tag in the format `@issue:MSP-2` then "issue" should not be a reserved word.

In other words, the pattern matching in [this code](https://github.com/serenity-js/serenity-js/blob/master/packages/core/src/model/tags/Tags.ts#L17) should also require the presence of a colon `:` character after "issue" to trigger the Jira integration.

**Comment by jan-molak** [comment]
Hi @sarahgoldman and thanks for reporting this. What version of Serenity/JS are you using, please?

**Comment by sarahgoldman** [comment]
I am using 2.24.1 in my dependency tree as part of cucumber 7.0.0 and protractor 7.0.0.

**Comment by jan-molak** [comment]
I believe this issue should have been fixed in [`2.25.3`](https://serenity-js.org/changelog#2-25-3-2021-03-20-)
Could you please upgrade to the latest version, `2.29.6`, and let me know if addresses the problem you're seeing?

**Comment by sarahgoldman** [comment]
Updated to 2.29.7 and the issue is resolved, thanks!

**Comment by jan-molak** [comment]
Fab, thanks for confirming :-) Enjoy Serenity!
